CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+First of all, thanks for contributing!
+
+This document provides some basic guidelines for contributing to this repository. To propose improvements, feel free to submit a pull request.
+
+## Submitting issues
+
+GitHub issues are welcome, feel free to submit error reports and feature requests.
+
+- Ensure the bug was not already reported by searching on GitHub under [Issues](https://github.com/DataDog/synthetics-ci-github-action/issues).
+- If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/DataDog/synthetics-ci-github-action/issues/new/choose).
+- Make sure to add enough details to explain your use case.
+
+If you require further assistance, contact [Datadog Support](https://docs.datadoghq.com/help/).
+
+## Submitting pull requests
+
+Have you fixed a bug or wrote a new feature and want to share it? Thanks!
+
+To expedite your pull request's review, follow these best practices when submitting your pull request:
+
+- **Write meaningful commit messages**: Messages should be concise and explanatory. The commit message describes the reason for your change, which you can reference later.
+
+- **Keep it small and focused**: Pull requests should contain only one fix or one feature improvement. Bundling several fixes or features in the same PR makes it more difficult to review, and takes longer to release.
+
+- **Write tests for the code you wrote**: Each script should be tested. The tests for a script are located in the [`src/tests` folder](https://github.com/DataDog/synthetics-ci-orb/tree/main/src/tests), under a file with the same name as the script. 
+**Note:** Datadog's internal CI is not publicly available, so if your pull request status is failing, make sure that all test pass locally. The Synthetics team should address errors flagged by the CI.
```

Comment:
@alai97 I don't know if we want to call this out but this is the CI we're referring to

Reply:
```suggestion

**Note:** Datadog's internal CI is not publicly available, so if your pull request status is failing, make sure that all tests pass locally. The Synthetics team can help you address errors flagged by the CI.
```